browser/src/control/Control.JSDialog.js

```diff
@@ -706,8 +710,10 @@ L.Control.JSDialog = L.Control.extend({
 				instance.updatePos();
 			}
 
-			if (instance.isAutofilter)
+			if (instance.isAutofilter && instance.id !== 'autoFillPreviewTooltip')
 				this.calculateAutoFilterPosition(instance);
+			else if (instance.id === 'autoFillPreviewTooltip')
```

Comment:
can be `instance.isAutoCompletePopup`?

Reply:
`instance.isAutoFillPreviewTooltip` is used here.
Done.